exercises/12-Delete-element/README.es.md

```diff
@@ -1,12 +1,38 @@
 # `12` Eliminar el elemento
 
-La única forma de eliminar a `Daniella` del array o arreglo (sin hacer trampa) será crear un nuevo arreglo con todas las demás personas, excepto Daniella.
+La única forma de eliminar a `Daniella` del array o arreglo (sin hacer trampa) es crear un nuevo arreglo con todas las demás personas, excepto `Daniella`.
+
+
+Así se comporta el método  `array.filter()`, deberías saberñp. Similar a los métodos `array.forEach()` y `array.map()`, es una función de orden superior, lo que significa que llama a otra función para lograr sus objetivos. 
+
+Esa funcion **callback** (de retorno) se llama `array.filter()` que acepta hasta 3 parámetros (opcionales) y el valor devuelto solo puede ser una cosa - una condición:
+
+```js
+
+(elementBeingIterated, indexOfThatElement, theIteratedArray) => condition;
+
+```
+Asi que quieres quedarte solo con los números 2 y 4 del array o arreglo de números, tu método `array.filter()`se vería de esta forma:
```

Comment:
Add a space between the words: `array.filter()`se, like this: `array.filter()` se